Documentation/CRDs/Object-Storage/ceph-object-store-crd.md

```diff
@@ -190,29 +190,35 @@ The gateway settings correspond to the RGW daemon settings.
     for more details. Multiple endpoints can be given, but for stability of ObjectBucketClaims, we
     highly recommend that users give only a single external RGW endpoint that is a load balancer that
     sends requests to the multiple RGWs.
+
+    Example of external rgw endpoints to connect to:
+
+    ```yaml
+    gateway:
+    port: 80
+    externalRgwEndpoints:
+      - ip: 192.168.39.182
+        # hostname: example.com
+    ```
+
 * `annotations`: Key value pair list of annotations to add.
 * `labels`: Key value pair list of labels to add.
 * `placement`: The Kubernetes placement settings to determine where the RGW pods should be started in the cluster.
 * `resources`: Set resource requests/limits for the Gateway Pod(s), see [Resource Requirements/Limits](../Cluster/ceph-cluster-crd.md#resource-requirementslimits).
 * `priorityClassName`: Set priority class name for the Gateway Pod(s)
+* `additionalVolumeMounts`: additional volumes to be mounted to the RGW pod. The root directory for
```

Comment:
Shall we add an example to the doc? Or maybe the example in object.yaml is sufficient.

Reply:
I was debating about that yesterday. I think this will be a pretty infrequently used setting, and an example here seemed like it might just add complexity for very few users' benefit. I'm hoping that mentioning the `subPath` below and how that appears in the container, along with the `object.yaml` example will be sufficient.

I thought it might be nice to link to the section in https://rook.io/docs/rook/latest-release/CRDs/specification. That seems like it would provide the best user help without being long-winded, but I can't make that link as a yaml file reference because the spec is generated as HTML. I would have to just add what an external-looking link that looks like `https://rook.io/docs/rook/latest-release/CRDs/specification/#ceph.rook.io/v1.CephObjectStore....`.

Reply:
Agreed this setting is going to be advanced/infrequent enough we don't need more examples in the doc. I'm fine with just the object.yaml example.